spec/features/connect_twitter_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe "connect a user" do
+  context "with valid params" do
+
+    before do
+      visit "/users/new"
```

Comment:
There is a `login_as(user)` helper in `spec/support/authentication.rb` that does all this for us. Just an easy way to clean up code!